### PR TITLE
onlykey-cli: init 1.2.2

### PR DESCRIPTION
**Pull request by ranfdev**
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7107,6 +7107,10 @@
     githubId = 14829269;
     name = "Ram Kromberg";
   };
+  ranfdev = {
+    email = "ranfdev@gmail.com";
+    name = "Lorenzo Miglietta";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:7112]: You need to set your github name and id.

File: pkgs/tools/security/onlykey-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  pname = "onlykey-cli";
+  version = "1.2.2";
+
+  src = python3Packages.fetchPypi {
+    inherit version;
+    pname = "onlykey";
+    sha256 = "1qkbgab5xlg7bd0jfzf8k5ppb1zhib76r050fiaqi5wibrqrfwdi";
+  };
+
+  # Requires onlykey
```
Comment (SuperSandro2000) [inline pkgs/tools/security/onlykey-cli/default.nix:13]: You mean the server or daemon application? Can this easily be added to nix in the future? You don't need to do it now, just curious.

File: pkgs/tools/security/onlykey-cli/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  doCheck = false;
+  propagatedBuildInputs =
+    with python3Packages; [ hidapi aenum six prompt_toolkit pynacl ecdsa cython ];
+
+  meta = with lib; {
+    description = "OnlyKey client and command-line tool";
+    homepage = "https://github.com/trustcrypto/python-onlykey";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ranfdev ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/onlykey-cli/default.nix:22]: Does this work on all platforms?

File: pkgs/tools/security/onlykey-cli/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{ lib, python3Packages, stdenv }:
```
Comment (SuperSandro2000) [inline pkgs/tools/security/onlykey-cli/default.nix:1]:
```suggestion
{ lib, python3Packages }:
```

stdenv is not actually used.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/security/onlykey-cli/default.nix (hunk 2 above)
Comment (ranfdev) [reply pkgs/tools/security/onlykey-cli/default.nix:13]: ```# Requires onlykey``` means: requires having the physical onlykey (a usb security key)

File: pkgs/tools/security/onlykey-cli/default.nix (hunk 3 above)
Comment (ranfdev) [reply pkgs/tools/security/onlykey-cli/default.nix:22]: It's in python and on the onlykey website they say it's compatible for linux, mac and windows. I can't test it for mac os though.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (ranfdev) [reply maintainers/maintainer-list.nix:7112]: Ok, Done

Comment (SuperSandro2000) [reply maintainers/maintainer-list.nix:7112]: Please format your message like ``maintainers: add name``.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102674` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>onlykey-cli</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102674` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>onlykey-cli</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (ranfdev) [reply maintainers/maintainer-list.nix:7112]: I've renamed the commit as requested

**Pull request merged**